### ISSUE TITLE
Feature Request: Auto reload .sym files.

### ISSUE THREAD

**Issue by basalicante** [openMSX/openMSX#1668]
Would it be possible to automatically reload any active .sym files if they are changed on disk?

Alternatively, maybe it's possible to add a checkbox on the Symbol Manager window to activate a reload button on the Debugger tool bar.

My use case is that when I'm disassembling, I like to run openMSX on the side so I can see what's going on. My disassembler outputs the .sym file, but openMSX doesn't refresh. The workaround would be to always keep the Symbol Manager window open.

It's definitely not high priority. I might be the only one using this ;-)

**Comment by m9710797** [comment]
I agree. Ideally we'd implement (optional) auto-refresh when the symbol file changes.
The practical problem is that this needs platform specific code (to get an OS notification when a watched file changes). I know how to do this for Linux. but not for Windows or MacOS (or any other platforms).
Because of this difficulty and because it's not super important, this has never been near the top of my TODO list.

**Comment by basalicante** [comment]
Yeah, I guessed it would be OS related.

The alternative would be to have a button in a handy place and manually updating.
Or if not, don't worry, it can also go on the "nice to have one day" list.
